### PR TITLE
Add project model

### PR DESCRIPTION
**Pull request by arturs105**

### CODE REVIEW

File: Toggl.Ultrawave/Models/Project.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+namespace Toggl.Ultrawave
+{
+    public sealed class Project : IProject
+    {
+        public int Id { get; set; }
+
+        [JsonProperty("wid")]
+        public int WorkspaceId { get; set; }
+
+        [JsonProperty("cid")]
```
Comment (amulware) [inline Toggl.Ultrawave/Models/Project.cs]: Are there no `workspace_id` and `client_id` fields in the api?

Comment (arturs105) [inline Toggl.Ultrawave/Models/Project.cs]: Nope, it has `wid` and `cid`: https://github.com/toggl/toggl_api_docs/blob/master/chapters/projects.md

`toggl.com/api/v9/me/projects` also returns `wid` and `cid`

**Comment by willsb** [comment]
So, is this merge ready, then? Is backend going to add the longer versions of the fields or are we using `wid` etc?

**Comment by arturs105** [comment]
I think we should merge this for now, since this is needed for `ProjectsApi`. We can simply delete the `JsonProperty` attributes, when backend adds long versions.

**Pull request merged**